apps/server/src/modules/board/controller/api-test/board-drawing-child.api.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { EntityManager, ObjectId } from '@mikro-orm/mongodb';
+import { ICurrentUser } from '@modules/authentication';
+import { JwtAuthGuard } from '@modules/authentication/guard/jwt-auth.guard';
+import { ServerTestModule } from '@modules/server/server.module';
+import { ExecutionContext, INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { ApiValidationError } from '@shared/common';
+import { BoardExternalReferenceType } from '@shared/domain/domainobject';
+import {
+	cardNodeFactory,
+	cleanupCollections,
+	columnBoardNodeFactory,
+	columnNodeFactory,
+	courseFactory,
+	mapUserToCurrentUser,
+	userFactory,
+} from '@shared/testing';
+import { Request } from 'express';
+import request from 'supertest';
+import { drawingElementNodeFactory } from '@shared/testing/factory/boardnode/drawing-element-node.factory';
+
+const baseRouteName = '/boards';
+
+class API {
+	app: INestApplication;
+
+	constructor(app: INestApplication) {
+		this.app = app;
+	}
+
+	async get(id: string) {
+		const response = await request(this.app.getHttpServer())
+			.get(`${baseRouteName}/${id}/hasDrawingChild`)
+			.set('Accept', 'application/json');
+
+		return {
+			result: response.text === 'true',
+			error: response.body as ApiValidationError,
+			status: response.status,
+		};
+	}
+}
+
+describe(`has drawing child (api)`, () => {
+	let app: INestApplication;
+	let em: EntityManager;
+	let currentUser: ICurrentUser;
+	let api: API;
+
+	beforeAll(async () => {
+		const module: TestingModule = await Test.createTestingModule({
+			imports: [ServerTestModule],
+		})
+			.overrideGuard(JwtAuthGuard)
+			.useValue({
+				canActivate(context: ExecutionContext) {
+					const req: Request = context.switchToHttp().getRequest();
+					req.user = currentUser;
+					return true;
+				},
+			})
+			.compile();
```

Comment:
we don't use overwriteGuard and use value anymore, but can import TestApiClient instead
check board-update-title.api.spec for example

Reply:
done